docs/v3/develop/write-tasks.mdx

```diff
@@ -236,10 +236,10 @@ list(generator) # prints 'Generator consumed!'
 
 ## Concurrency
 
-Tasks enable concurrent execution, allowing you to execute multiple tasks asynchronously.
+Tasks enable concurrent execution, allowing you to execute multiple tasks concurrently.
 This concurrency can greatly enhance the efficiency and performance of your workflows.
 
-Expand the script to calculate the average open issues per user by making more requests:
```

Comment:
just "Expand the script" is out of left field since this is not nested in the earlier section that contains the example